playbooks/system-criticality.md

```diff
@@ -31,21 +31,23 @@ Retired systems, spikes, or time-bound experiments that aren't significantly use
+| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Development       | <ul><li>READMEs contain up-to-date set-up instructions</li><li>Code review on all [pull requests](/playbooks/engineer-workflow.md#pull-requests)</li><li>New systems or significant new components undergo [tech review]()</li></ul> | <ul><li>&larr;ditto</li></ul>                                                            | <ul><li>&larr;ditto and...</li><li>Production environment is replicable locally (TODO)</li></ul>                                                                       |
+| Testing           | <ul><li>Automated tests</li><li>Tests are run on PRs and `master` by a Continuous Integration pipeline</li></ul>                                                                                                                     | <ul><li>&larr;ditto</li></ul>                                                            | <ul><li>&larr;ditto and...</li><li>Test coverage tooling</li></ul>                                                                                                     |
+| Deployment        | <ul><li>Zero-downtime deployment</li></ul>                                                                                                                                                                                           | <ul><li>&larr;ditto and...</li><li>High-fidelity staging environment</li></ul>           | <ul><li>&larr;ditto and...</li><li>Deployment and orchestration by Kubernetes</li></ul>                                                                                |
+| Performance       |                                                                                                                                                                                                                                      |                                                                                          | <ul><li>Latency-based monitors and alerting (tailored to service)</li></ul>                                                                                            |
```

Comment:
Would add here at a minimum:
- Error rate
- P90 latency

Datadog prompts you to set these up when creating a new service

Optionally:
- % change in throughput
- P75 latency for more critical apps